profile_bluesky/startup/instrument/devices/util_components.py

```diff
@@ -149,23 +149,10 @@ def __init__(self, prefix="", *, readback_pv="", setpoint_pv="",
     def _setup_move(self, position):
```

Comment:
How does this differ from the `set()` method?

Reply:
See previous comment.